modules/creating-microshift-development-environment.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
```

Comment:
We probably want to call this something other than the "development environment." These instructions are a good way to prepare a VM for experimenting with MicroShift, so if we can use some of that language that would be better.